src/ripple/app/ledger/LedgerHistory.cpp

```diff
@@ -517,7 +517,7 @@ bool LedgerHistory::fixIndex (
     return true;
 }
 
-void LedgerHistory::tune (int size, int age)
+void LedgerHistory::tune (int size, std::chrono::seconds age)
```

Comment:
Suggestion: `(int size, seconds age)`

Reply:
I've instead removed the `using` per Scott's recommendation.

Reply:
👍 I prefer @scottschurr's recommendation.